src/screens/About/Team/TeamSection.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment } from 'react';
+import { teams, teamsWithTitles } from 'cms-content/team';
+import { AboutHeading3 } from '../About.style';
+import { sortBy } from 'lodash';
+import Grid from '@material-ui/core/Grid';
+import ActiveMember from './ActiveMember';
+import AlumniMember from './AlumniMember';
+
+const TeamSection = () => {
+  return (
+    <Fragment>
+      {teams.map(team => {
+        const alphabetizedTeam = sortBy(
+          team.teamMembers,
+          member => member.fullName,
+        );
+        const teamToMap =
+          team.teamName === 'Leadership' ? team.teamMembers : alphabetizedTeam;
+        const isAlumni = team.teamName === 'Alumni';
+        const includeTitle = teamsWithTitles.includes(team.teamName);
+        return (
+          <Fragment>
```

Comment:
Does this need to be a fragment over a div or grid container?

Reply:
this fragment contains the grid container (which contains a grid item, in this case `AlumniMember` or `ActiveMember`). react doesnt let you return adjacent elements (in this case `AboutHeading3` and a `Grid` container) without wrapping them in a parent element--the fragment acts as the parent without adding anything to the dom.